src/index.js

```diff
@@ -1,27 +1,32 @@
-import _ from 'lodash';
 import './style.css';
 
 const container = document.getElementById('container_ul');
 
 const list = [{
-    description: 'wash the dishes',
-    completed: false,    
+  description: 'wash the dishes',
+  completed: false,
+  index: 1,
 }, {
-    description: 'buy milk',
-    completed: false, 
+  description: 'buy milk',
+  completed: false,
+  index: 2,
 }, {
-    description: 'clean the floor',
-    completed: false, 
-}
+  description: 'clean the floor',
+  completed: false,
+  index: 3,
+},
 ];
-
+console.log(list.length);
```

Comment:
- [ ] Kindly remove this console statement: they make your code look messy. Always double-check to ensure that you remove all console log statements in the future before you commit your changes.